src/rules/order.ts

```diff
@@ -824,6 +825,7 @@ function getRequireBlock(node: TSESTree.Node) {
 const types: ImportType[] = [
   'builtin',
   'external',
+  'private-imports',
```

Comment:
```suggestion
  'private-import',
```

Oh, sorry, to be consistent with other types, we should use singular here.